src/components/button/Button.ts

```diff
@@ -0,0 +1,23 @@
+import Block from '../../utils/Block';
+import template from './template.hbs';
+import './style.less';
+
+export interface ButtonProps {
+    id: string;
+    value?: string;
+    classes?: string;
+    type: string;
```

Comment:
тип кнопки может принимать ограниченное количество значений, в него нельзя положить любую строку. Предлагаю воспользоваться множеством
```suggestion
    type: 'button' | 'submit' и тд;
```